back-end/routes/index.js

```diff
@@ -4,6 +4,10 @@ var router = express.Router();
 
 /* GET home page. */
 router.get('/', function(req, res, next) {
+  if (process.env.NODE_ENV === 'dockerlocal') {
```

Comment:
Since there's no build folder during development, this just redirects the user back to the port the React app is being served on.